### PR TITLE
fix: [CVNG-4122]: Validation for prometheus identifier duplicate 

### PR DESCRIPTION
**Pull request by praneshg239**
##### Summary:

image

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by praneshg239** [comment]
retrigger titlecheck

File: src/modules/85-cv/pages/health-source/connectors/PrometheusHealthSource/tests/PrometheusHealthSource.test.tsx [hunk 1]
```diff
@@ -235,6 +236,36 @@ describe('Unit tests for PrometheusHealthSource', () => {
...
+      userEvent.click(container.querySelector('button[class*="manualQuery"]')!)
+    })
+    await waitFor(() => expect(getByText('cv.monitoringSources.prometheus.isManualQuery')).not.toBeNull())
+    expect(container.querySelectorAll('[class*="Accordion--panel"]').length).toBe(2)
+
+    act(() => {
+      userEvent.click(getByText('submit'))
+    })
+
+    // expect(screen.getByText(/validation.identifierRequired/i)).toBeInTheDocument()
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/connectors/PrometheusHealthSource/tests/PrometheusHealthSource.test.tsx:258]: remove

**Review by deepesh-ui — APPROVED** [review]

**Comment by praneshg239** [comment]
retrigger titlecheck

File: src/modules/85-cv/pages/health-source/connectors/PrometheusHealthSource/tests/PrometheusHealthSource.test.tsx [hunk 2]
```diff
@@ -163,9 +164,9 @@ describe('Unit tests for PrometheusHealthSource', () => {
     await waitFor(() => expect(container.querySelector('input[name="sli"')).toBeInTheDocument())
 
     // Correct warning message is shown
-    await waitFor(() =>
-      expect(getByText('cv.monitoringSources.gco.mapMetricsToServicesPage.validation.baseline')).not.toBeNull()
-    )
+    // await waitFor(() =>
+    //   expect(getByText('cv.monitoringSources.gco.mapMetricsToServicesPage.validation.baseline')).not.toBeNull()
+    // )
```
Comment (praneshg239) [inline src/modules/85-cv/pages/health-source/connectors/PrometheusHealthSource/tests/PrometheusHealthSource.test.tsx:169]: Did on purpose, @deepesh-ui please check this

**Review by deepesh-ui — APPROVED** [review]

**Review by shashidhargg — APPROVED** [review]

**Pull request merged**